man/systemd.network.xml

```diff
@@ -796,6 +796,52 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[IPRule] Section Options</title>
+
+      <para>An <literal>[IPRule]</literal> section accepts the
+      following keys. Specify several <literal>[IPRule]</literal>
+      sections to configure rules in the routing policy database control the
+       route selection algorithm. </para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>TypeOfService=</varname></term>
+          <listitem>
+            <para> Specifies type of service.</para>
```

Comment:
Spurious space.

This doesn't really explain anything — it just repeats the name of the option. Please say something about possible values and what they mean.